app/controllers/users/omniauth_callbacks_controller.rb

```diff
@@ -1,5 +1,6 @@
 class Users::OmniauthCallbacksController < Devise::OmniauthCallbacksController
   def login_dot_gov
+    @kind = "Login.gov"
```

Comment:
I looked at `auth_hash["provider"]` for this info, but it returns `github` and `login_dot_gov`. I think `Login.gov` and `GitHub` is better, per what you have. 👍